assets/app/views/attachPVC.html

```diff
@@ -1,172 +1,186 @@
-<div class="content">
-  <div class="container">
-    <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
-    <div class="row">
-      <div class="col-md-12">
-        <alerts alerts="alerts"></alerts>
-        <div ng-show="!pvcs || !(attach.deploymentConfig || attach.deployment)">Loading...</div>
+<default-header class="top-header"></default-header>
+<div class="wrap no-sidebar">
+  <div class="sidebar-left collapse navbar-collapse navbar-collapse-2">
+    <navbar-utility-mobile></navbar-utility-mobile>
+  </div>
 
-        <div ng-show="pvcs && !pvcs.length && (attach.deploymentConfig || attach.deployment)" class="empty-state-message empty-state-full-page">
-          <h2 class="text-center">No persistent volume claims.</h2>
+  <div class="add-to-project middle surface-shaded">
+    <!-- Middle section -->
+    <div class="middle-section">
+      <div class="middle-container has-scroll">
+        <div class="middle-content">
+          <div class="container">
```

Comment:
`.middle .middle-section .middle-container .middle-content .container` is a lot of nesting with very similar names.  I wonder if names like `.middle-scollable-panel` (though a bit longer) would help clarify intent.  

I am a bit uneasy about directives only for dealing with nesting (like `project-page`).  Wondering if we can use includes better to breakdown templates overall.